WalletWasabi/Logging/Logger.cs

```diff
@@ -46,7 +46,7 @@ public static class Logger
 		/// Gets or sets the maximum log file size in KB.
 		/// </summary>
 		/// <remarks>Default value is approximately 10 MB.</remarks>
-		private static long MaximumLogFileSize { get; set; } = 10_000;
+		private static long MaximumLogFileSize { get; set; } = MinimumLevel == LogLevel.Trace ? 50_000 : 10_000;
```

Comment:
What about putting it to https://github.com/zkSNACKs/WalletWasabi/blob/29f0201916cf840c65548c77cc2c9fcb7f694fb1/WalletWasabi/Logging/Logger.cs#L80?